### PR TITLE
Replace slack references with Discord

### PR DESCRIPTION
**Pull request by harshitsinghai77**
Deta Slack workplace has been deleted.

This PR replaces all slack references to discord.

### CODE REVIEW

File: docs/home.md [hunk 1]
```diff
@@ -78,5 +78,5 @@ You create and update micros using the CLI.
 We have answered some [FAQs](faqs.md) in these docs, but we are here to help and would love to hear what you think!
 
 If you have any questions or feedback, you can reach us:
-- [Slack](https://join.slack.com/t/deta-hq/shared_invite/zt-ej8ygys5-1szO9l~y052Hg04FXdV1iA)
+- [Discord](https://discord.gg/cGZV5Shc)
```
Comment (abdelhai) [inline docs/home.md]:
```suggestion
- [Discord](https://go.deta.dev/discord)
```

File: docs/micros/custom_domains.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ Check the following if you are experiencing issues in setting up a custom domain
 3. Check if there is a CAA record set up for your domain. If yes, make sure you have [set up the necessary CAA record](#caa-records).
 
 :::warning
-If you had to change something from the steps mentioned above, [email us](<mailto:aavash@deta.sh?subject=Re-enable custom domain>) or [let us know in slack](https://deta.hq.slack.com). This is important as our systems eventually stop trying to assign the custom domain on errors. Manual re-enabling of the domain might be required.
+If you had to change something from the steps mentioned above, [email us](<mailto:aavash@deta.sh?subject=Re-enable custom domain>) or [let us know in discord](https://discord.gg/cGZV5Shc). This is important as our systems eventually stop trying to assign the custom domain on errors. Manual re-enabling of the domain might be required.
```
Comment (abdelhai) [inline docs/micros/custom_domains.md]:
```suggestion
If you had to change something from the steps mentioned above, [email us](<mailto:aavash@deta.sh?subject=Re-enable custom domain>) or [let us know in discord](https://go.deta.dev/discord). This is important as our systems eventually stop trying to assign the custom domain on errors. Manual re-enabling of the domain might be required.
```

File: docusaurus.config.js [hunk 3]
```diff
@@ -39,8 +39,8 @@ module.exports = {
           title: 'Community',
           items: [
             {
-              label: 'Slack',
-              href: 'https://deta-hq.slack.com/',
+              label: 'Discord',
+              href: 'https://discord.gg/cGZV5Shc',
```
Comment (abdelhai) [inline docusaurus.config.js]:
```suggestion
              href: 'https://go.deta.dev/discord',
```

**Review by abdelhai — CHANGES_REQUESTED** [review]
hi, thanks for the pr! just used the canonical url instead of a direct one

**Comment by harshitsinghai77** [comment]
Done.

**Review by abdelhai — APPROVED** [review]
thank you! @harshitsinghai77

**Comment by harshitsinghai77** [comment]
@abdelhai when do approve PRs gets merged?

**Comment by abdelhai** [comment]
hi @harshitsinghai77 apology, totally missed that merge button XD

**Pull request merged**